internal/testing/dwarftestdata/data.go

```diff
@@ -13,9 +13,6 @@ import (
 // line number, source code file, etc, can change. Therefore,
 // even though these binaries are huge, we check them in to the repositories.
 
-//go:embed testdata/tinygo/main.wasm
-var TinyGoWasm []byte
```

Comment:
decided to delete DWARF test case with TinyGo as I found that it's not reliable. https://github.com/tinygo-org/tinygo/issues/3444